package.json

```diff
@@ -17,12 +17,22 @@
     "script": "./dist/index.js",
     "default": "./dist/main.js"
   },
+  "scripts": {
+    "build": "npm-run-all clean build:js build:go",
+    "build:js": "rollup -c",
+    "build:go": "cd cmd && GOOS=js GOARCH=wasm go build -ldflags=\"-s -w\" -o ../dist/excelize.wasm && cd ../dist && gzip -9 -v -c excelize.wasm > excelize.wasm.gz && rm excelize.wasm",
+    "clean": "rm -rf dist",
+    "test": "node test/test-nodejs.js"
+  },
```

Comment:
Any of these scripts can be executed by running `npm run x` - e.x. `npm run build`.
`npm-run-all` is a library that allows running multiple npm commands without having to chain them together.

`gzip -9 -v -c excelize.wasm > excelize.wasm.gz && rm excelize.wasm` is probably not cross-platform compatible, likely won't work on Windows.